tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\Manager::executeWriteCommand() throws CommandException for invalid writeConcern
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('REPLICASET'); CLEANUP(REPLICASET); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$document = ['foo' => ['bar']];
+
+$command = new MongoDB\Driver\Command([
+    'findAndModify' => COLLECTION_NAME,
+    'query' => ['_id' => 'foo'],
+    'update' => $document,
+    'upsert' => true,
+    'new' => true,
+]);
+
+try {
+    $manager->executeWriteCommand(DATABASE_NAME, $command, ['writeConcern' => new MongoDB\Driver\WriteConcern("minority")]);
```

Comment:
Since this is an intentionally undefined mode, I'll suggest using `'undefined'` here. This is similar enough to the actual "majority" mode that it might confuse someone that comes across the test later.